### PR TITLE
Move cleanup before build (v2)

### PR DESCRIPTION
**Pull request by joonas-fi**

### CODE REVIEW

File: builder/packaging/package.sh [hunk 1]
```diff
@@ -111,39 +104,30 @@ if [ -e ${mod_dir}/deps_ws ]; then
...
 # --parallel flag is needed in "fakeroot debian/rules binary" call.
 export DEB_BUILD_OPTIONS="parallel=`nproc`"
 
-echo "[INFO] Clean up."
+# generates makefile at debian/rules, which is used to invoke the actual build.
+bloom-generate rosdebian --os-name ubuntu --os-version focal --ros-distro ${ROS_DISTRO} --place-template-files
+
+sed -i "s/@(DebianInc)@(Distribution)/@(DebianInc)/" debian/changelog.em
+
+[ ! "$distr" = "" ] && sed -i "s/@(Distribution)/${distr}/" debian/changelog.em || :
```
Comment (joonas-fi) [inline builder/packaging/package.sh:117]: Please review this line carefully. The syntax at end is unfamiliar to me, but I tested it in my shell and it seemed to work.

**Review by mehmetkillioglu — APPROVED** [review]
Tested with mocap_pose and rplidar_ros2, works fine.

File: builder/packaging/package.sh (hunk 1 above)
Comment (maseabunikie) [reply builder/packaging/package.sh:117]: This syntax is ok.

**Review by maseabunikie — APPROVED** [review]
Ship it!

**Pull request merged**